sklearn/metrics/classification.py

```diff
@@ -1068,8 +1068,7 @@ def fbeta_score(y_true, y_pred, beta, labels=None, pos_label=1,
 
     The `beta` parameter determines the weight of precision in the combined
     score. ``beta < 1`` lends more weight to precision, while ``beta > 1``
-    favors recall (``beta -> 0`` considers only precision, ``beta -> inf``
-    only recall).
+    favors recall (``beta -> inf`` considers only recall).
```

Comment:
I don't think there's anything wrong with `beta -> 0` personally, but it means "the limit as beta approaches 0", and this may confuse users. Just get rid of the stuff in the parentheses, and instead say `0 < beta < 1 lends more weight to precision` above to express that the parameter is bounded.

Reply:
Indeed, my commit turned out to be more confusing and even misleading, I will change it towards making it sound clearer

Reply:
want to still fix this?

Reply:
Yes @amueller,

I will correct it

Reply:
indeed, thanks :)